spark-gremlin/src/test/java/org/apache/tinkerpop/gremlin/spark/SparkGraphFeatureIntegrateTest.java

```diff
@@ -162,6 +162,11 @@ private static Map<String, Object> getBaseConfiguration(final GraphData graphDat
                 put(Constants.SPARK_KRYO_REGISTRATION_REQUIRED, true);
             }};
         }
+
+        @Override
+        public boolean useParametersLiterally() {
+            return false;
```

Comment:
as noted for TinkerGraph, I feel like we need to test both configurations because we want to test `GValue` (when this is "false") and straight translation (when this is "true"). feels like we're leaving something out by not having both configurations.

Reply:
I opted to leave Spark and Hadoop with the single configuration as these suites are quite slow to run and I don't see much value in running both configurations. I can make the change if you believe the added coverage is needed.